core/src/main/java/org/mule/runtime/core/internal/context/DefaultMuleContext.java

```diff
@@ -318,6 +319,7 @@ public class DefaultMuleContext implements MuleContextWithRegistry, PrivilegedMu
   private ConfigurationComponentLocator componentLocator;
 
   private LifecycleStrategy lifecycleStrategy = new DefaultLifecycleStrategy();
+  private Level splashScrenLevel = Level.INFO;
```

Comment:
Use static imports as specified in the [coding conventions](https://docs.google.com/document/d/1N2Axq3ABJjYmj9BjfYQkvJfF0odHQnlb8MAdVQydOWI/edit#heading=h.ujxk1qbe2hxj).